### DISCUSSION TITLE
Timeliness of Rates For Inclusion in a 1/1 data file

### DISCUSSION THREAD

**Discussion by ryan-thomas-peterson** [CMSgov/price-transparency-guide#643] [Q&A] [answered]
Many contracted rates are based upon federal fee schedules which supply a final dollar amount.  Some of those fee schedules do not get finalized until days before the new year (or what appears likely here this year will be updated shortly after the new year).   Specifically, legislation passing congress today to be signed in the next week would change the CMS RBRVS rates that are active on January 1st, 2023 but data producers don't have that information yet.  Is there guidance for how producers of these data should handle this?

Should we, delay the production of a 1/1 file?  
Produce a 1/1 data file with disclaimers that rates will be updated in the next monthly delivery?  
Something else?

How are other producers handling this?

**Comment by JR-CMS** [comment, marked as answer]
The Transparency in Coverage final rules require the machine-readable files to be updated monthly. The group health plan or health insurance issuer must clearly indicate the date that the files were most recently updated. The Departments consider “monthly” to refer to reasonably consistent periods of approximately 30 days but are not specifying a particular day of the month. 

The content requirements for the machine-readable files, as detailed in 54.9815-2715A3(b)(1)(i)(C), 2590.715-2715A3(b)(1)(i)(C), and 147.212(b)(1)(i)(C) require the inclusion of all applicable rates, including underlying fee schedule rates. These rates must be associated with the **last date of the contract term or expiration date** for each provider-specific applicable rate that applies to each covered item or service. 

As the above bold section indicates, the machine-readable files for January 2023 should be accurate and include rates through the last date of the contract term or expiration date for the January 2023 files’ reporting period. If the CMS RBRVS rates are not publicly available for the January 2023 files’ reporting period, you should report the rates associated with the last date of the contract term.

**Reply by tdfow** [reply]
@shaselton-usds @JR-CMS Can you confirm the above is the same for evergreen contracts, since evergreen contracts do not have a set expiration date?

**Reply by JR-CMS** [reply]
The above applies to all contract types. 

We also discussed evergreen contracts in this post here: https://github.com/CMSgov/price-transparency-guide/discussions/42.
